src/platform/shared.c

```diff
@@ -538,6 +512,34 @@ void platform_process_messages()
 				break;
 			}
 			break;
+// Apple sends touchscreen events for trackpads, so ignore these events on OS X
+#ifndef __MACOSX__
```

Comment:
I've never seen this macro before, are you sure that it works? In the other files we are using `#if defined(__APPLE__) && defined(__MACH__)`